SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -478,7 +478,42 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 		// Taken from PostNetInit
 		if (NetDriver->GetWorld()->HasBegunPlay() && !EntityActor->HasActorBegunPlay())
 		{
+			if (!NetDriver->IsServer())
+			{
+				check(!EntityActor->HasAuthority());
```

Comment:
This check could be combined with the server condition. check (is server || !authority)